hack/build-cross.sh

```diff
@@ -28,6 +28,7 @@ KUBE_BUILD_PLATFORMS=("${KUBE_SERVER_PLATFORMS[@]}")
 kube::golang::build_binaries "${KUBE_SERVER_TARGETS[@]}"
 
 KUBE_BUILD_PLATFORMS=("${KUBE_CLIENT_PLATFORMS[@]}")
+KUBE_STATIC_OVERRIDES=("kubectl")
```

Comment:
OK. Soon I'll look at simply running the build and publishing the static result up to the ```netexec``` container as part of the the test run. I'll then remove the forced static ```kubectl``` from the main build.